### DISCUSSION TITLE
How to configure a new page to function like the "projects" page?

### DISCUSSION THREAD

**Discussion by royabradley** [alshedivat/al-folio#650] [Q&A]
I am trying to create a "classroom" page that functions like the "projects" page (as hinted at in the [demo site](https://alshedivat.github.io/al-folio/teaching/): "You can convert it to a collection similar to `_projects/` so that you can have a dedicated page for each course.") 

I have duplicated the `projects.html` and `projects_horizontal.html` files and renamed them `classroom.html` and `classroom_horizontal.html` respectively, in the `_includes` directory. Then in the `_pages` directory I have created `classroom.md` and configured it exactly like `projects.md`, but changed every instance of "project" to "classroom" in the code. When I do this, I get the error:

```
 Liquid Exception: Liquid error (line 29): Cannot sort a null object. in _pages/classroom.md
```

I am obviously not configuring the page correctly, but I do not know the proper way to do this. Help is greatly appreciated.

**Comment by royabradley** [comment]
Any help with this?

**Comment by rohandebsarkar** [comment]
Most probably, the page `classroom.md` did not have a `title` or after 8db7daf35681d24b6597cd53f870ac81423b64ee it may not have a `nav_order` variable declared.

**Comment by wyhallenwu** [comment]
#568 https://github.com/alshedivat/al-folio/discussions/568#discussioncomment-5369810

**Comment by george-gca** [comment]
Duplicate of #568
